src/common/snippets/src/runtime_configurator.cpp

```diff
@@ -62,7 +64,7 @@ void RuntimeConfigurator::initialization(const lowered::LinearIRCPtr& linear_ir)
     m_latest_shapes.resize(m_io_num);
     m_config->io_data_offsets.resize(m_io_num);
     m_config->tile_rank = linear_ir->get_config().m_loop_depth;
-    m_optimizer.init(linear_ir, m_io_descs, m_in_num);
+    m_optimizer.init(linear_ir);
 
     // InnerSplittedLoops should be inited after OuterSplittedLoops
```

Comment:
It's really curiously 😄  It's redundant since we have `LoopInfo::apply` method now. Seems like I didn't remove this part when added the method. May I ask you to remove the code block with `m_ordered_loop_ids` please?

Reply:
I am always happy to remove something 😄
Done